src/app/pages/quickMaps/pages/baselineDetails/baselineDetails.component.html

```diff
@@ -3,8 +3,10 @@
   <app-base-est></app-base-est>
   <gridster-item [item]="widget" *ngFor="let widget of dashboard">
     <!-- Map View -->
-    <app-map-view [widget]="widget" [resizeEvent]="resizeEvent" *ngIf="widget.type === 'widgetMap'"></app-map-view>
-
+    <app-card2 #card [widget]="widget" [resizeEvent]="resizeEvent" *ngIf="widget.type === 'widgetMap'">
```

Comment:
when it all refactored are you planning to find+replace to something other than `app-card2`?

Reply:
I'll actually move the code from card2 to card to maintain the git history. :-)